### DISCUSSION TITLE
Use international date formats (DMY)

### DISCUSSION THREAD

**Discussion by getsnoopy** [rust-lang/crates.io#4445] [Feature Requests]
It looks like crates.io currently uses (e.g., on [the Downloads chart](https://crates.io/crates/parse_wiki_text) and [the Versions tab](https://crates.io/crates/parse_wiki_text/versions)) the esoteric and illogical MDY date formats (`Month DD` & `Month DD, YYYY`) that isn't used anywhere in the world except for the US (and some parts of Canada). Seeing as Rust is an international project, it only makes sense to use international standards such as the DMY date formats (`DD Month` & `DD Month YYYY`), as codified in the [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822#section-3.3) specification, which is used in the HTTP protocol itself, as well as any other major standard.

**Comment by PatchMixolydic** [comment]
This probably came about because the Rust project (or at least Cargo) by and large seems to prefer American English (for instance, [the command line option to control colour output](https://doc.rust-lang.org/cargo/commands/cargo-check.html#option-cargo-check---color) is spelled `--color`). Personally, though, I wouldn't really be bothered if the date format were changed (or retained).

Another option worth mentioning is the [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) date format (YYYY-MM-DD), which is another fairly well known international standard. Unfortunately, it doesn't use month names, which might negatively impact readability.

**Reply by getsnoopy** [reply]
I'm totally fine with ISO 8601 as well, and in fact, it would even be more preferable to the RFC 2822 format because it's language-agnostic and universally recognized. I only suggested RFC 2822 because the current format seemed to be a full-text format instead of an all-numeric format.

**Reply by Turbo87** [reply]
FWIW the current format is unintentional. We've implemented the rest of the page to use the formats provided by the browser locale, but the chart does not appear to use the same functionality yet. feel free to open a PR to fix it if you have the time :)
